doc/manual/manual.tex

```diff
@@ -8684,6 +8684,114 @@ \subsubsection{The Burstedde variable viscosity benchmark}
 \end{figure}
 
 
+\subsubsection{The hollow sphere benchmark}
+
+\label{sec:cookbooks-hollow-sphere}
+
+This benchmark is based on Thieulot [In prep.] in which an analytical solution to the
+isoviscous incompressible Stokes equations is derived in a spherical shell geometry.
+The velocity and pressure fields are as follows:
+
+\begin{eqnarray}
+v_r(r,\theta)      &=& g(r) \cos \theta \\
+v_\theta(r,\theta) &=& f(r) \sin \theta \\
+v_\phi(r,\theta)   &=& f(r) \sin \theta \\
+p(r,\theta)        &=& h(r) \cos \theta 
```

Comment:
add a comma to the end of each line (just like you would if these were formulas inlined in the text).